src/main/java/com/github/complate/nashorn/renderer/NashornComplateRenderer.java

```diff
@@ -20,9 +20,17 @@
 import java.util.Map;
 import java.util.Optional;
 
-import static java.util.Collections.emptyMap;
-import static javax.script.ScriptContext.ENGINE_SCOPE;
-import static jdk.nashorn.api.scripting.NashornException.getScriptStackString;
+import javax.script.Bindings;
```

Comment:
All these imports (line 23 to 33) are already present and are now duplicated. These should be removed.